Octokit.Tests/Exceptions/AbuseExceptionTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Octokit.Internal;
+using Xunit;
+
+namespace Octokit.Tests.Exceptions
+{
+    public class AbuseExceptionTests
```

Comment:
The style/verbage used in the naming of the test fixtures isn't too consistent with (some/most) of the other octokit tests.  

We normally word the test fixture naming with sub classes for the functions eg `TheConstructor` `TheParseRetryHeaderMethod` etc, and then the name of the `[Fixture]` test functions is like `HandlesEmptyHeader` `HandlesNullHeader` `CorrectlyParsesValue` and so on. 

So that way we end up with common english sounding

`AbuseExceptionTests=>TheConstructor=>CorrectlyDeterminesRetryAfterValue` 

and so on